src/frontend/apps/desk/src/features/menu/MenuItems.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button } from '@openfun/cunningham-react';
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+import React, { useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import { Box } from '@/components/';
+import { useCunninghamTheme } from '@/cunningham';
+import { SVGComponent } from '@/types/components';
+
+import { Tooltip } from './Tooltip';
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: #ffffff33;
+  &[aria-current='page'] {
+    color: #ffffff;
+  }
+`;
+
+interface MenuItemProps {
+  Icon: SVGComponent;
+  label: string;
+  href: string;
+  width?: number;
+}
+
+const MenuItem = ({ Icon, label, href, width = 38 }: MenuItemProps) => {
+  const { t } = useTranslation();
+  const pathname = usePathname();
+  const { colorsTokens } = useCunninghamTheme();
+  const buttonRef = useRef(null);
+  const [isTooltipOpen, setIsTooltipOpen] = useState(false);
+
+  const isActive = pathname === href;
+  const color = isActive ? colorsTokens()['primary-600'] : '#ffffff55';
+  const background = isActive ? colorsTokens()['primary-300'] : undefined;
```

Comment:
Should be the same.
```suggestion
  const background = isActive && colorsTokens()['primary-300'];
```

Reply:
`const background = isActive ? colorsTokens()['primary-300'] : undefined;` == `const background: string | undefined`
`const background = isActive && colorsTokens()['primary-300'];` == `const background: string | false`

Reply:
mybad it works, but triggers a typescript error.